back/db/migrate/20230517145937_update_que_tables_to_version5.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+class UpdateQueTablesToVersion5 < ActiveRecord::Migration[6.1]
+  def up
+    if Rails.env.development? || Rails.env.test?
+      # We need to run a slightly different migration in development and test
+      # environments because some objects that `que` expects to be present are very
+      # likely to be missing. This is because `schema.rb` cannot represent certain
+      # database constructs, such as triggers and functions. Therefore, a simple
+      # `db:reset` removes those objects.
+      Que.execute(DEV_QUE_SQL_MIGRATION_5)
+    else
+      Que.migrate!(version: 5)
+    end
+  end
+
+  def down
+    Que.migrate!(version: 4)
+  end
+
+  # We need to run a slightly different migration in development and test
+  # environments because some objects that `que` expects to be present are very
+  # likely to be missing. This is because `schema.rb` cannot represent certain
+  # database constructs, such as triggers and functions. Therefore, a simple
+  # `db:reset` removes those objects.
+  #
```

Comment:
Is the double comment intentional?

Reply:
yep! I wanted it close to both the Que.execute and the definition of the migration, but the file is so small I could remove one copy.